nodebuilder/state/keyring.go

```diff
@@ -15,24 +15,10 @@ type AccountName string
 // as having keyring-backend set to `file` prompts user for password.
 func Keyring(cfg Config, ks keystore.Keystore) (kr.Keyring, AccountName, error) {
 	ring := ks.Keyring()
-	var info *kr.Record
-	// if custom keyringAccName provided, find key for that name
-	if cfg.KeyringAccName != "" {
-		keyInfo, err := ring.Key(cfg.KeyringAccName)
-		if err != nil {
-			log.Errorw("failed to find key by given name", "keyring.accname", cfg.KeyringAccName)
-			return nil, "", err
-		}
-		info = keyInfo
-	} else {
-		// use default key
-		keyInfo, err := ring.Key(DefaultAccountName)
-		if err != nil {
-			log.Errorw("could not access key in keyring", "name", DefaultAccountName)
-			return nil, "", err
-		}
-		info = keyInfo
+	keyInfo, err := ring.Key(cfg.KeyringKeyName)
+	if err != nil {
+		log.Errorw("could not access key in keyring", "keyring.keyname", cfg.KeyringKeyName)
+		return nil, "", err
```

Comment:
You could still wrap it to add some context. You could make a new error to return and also pass it the log